liberty-maven-plugin/src/it/tests/test-server-param-default/pom.xml

```diff
@@ -41,6 +41,25 @@
                 <version>3.0.0</version>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>1.5</version>
+                <configuration>
+                    <target>
+                        <echo message="WLP_OUTPUT_DIR=${project.build.directory}/LibertyOutput" file="${project.basedir}/src/test/resources/server.env" />
+                    </target>
```

Comment:
Does this configuration goal overriding the project's server.env file? That's pretty slick for reusing project configuration.  Can you add a comment to this?

Reply:
Yes, it overrides it and I will add a comment to it.